cms/templates/js/move-xblock-list.underscore

```diff
@@ -35,6 +31,17 @@
                     <span class="icon fa fa-arrow-right forward-sr-icon" aria-hidden="true"></span>
                     <span class="sr forward-sr-text"><%- gettext("Click for children") %></span>
                 </button>
+            <% } else { %>
```

Comment:
I thought that we had previously changed "Click for children" to something else? Did it get lost in the previous PR?